cmd/cosign/cli/attest/attest.go

```diff
@@ -159,7 +159,7 @@ func AttestCmd(ctx context.Context, ko options.KeyOpts, regOpts options.Registry
 	}
 
 	// Check whether we should be uploading to the transparency log
-	if sign.ShouldUploadToTlog(ctx, ko, digest, force, noTlogUpload) {
+	if sign.ShouldUploadToTlog(ctx, ko, digest, force, tlogUpload) {
```

Comment:
what's the difference between this and `fource`?

Reply:
i believe the goal of force is to get around the user prompt that happens in `ShouldUploadToTlog`

https://github.com/sigstore/cosign/blob/1c15b03a6e9e47f5c85a8fd7ff98bf75e910df6a/cmd/cosign/cli/sign/sign.go#L88-L92

Reply:
We should rename force to be more descriptive

Reply:
i think we can just delete it in a separate PR, we have a `SkipConfirmation` field we can rely on instead

https://github.com/sigstore/cosign/blob/04f509c7c95538d50a6065c2f098eada59d9f139/cmd/cosign/cli/options/sign.go#L35

seems like the intention was to remove `--force` and replace it with that based on this comment back in July!

https://github.com/sigstore/cosign/pull/2039#discussion_r911985088

Reply:
Gotcha, cool: so `force` will be `skipConfirmation` then in another PR!